leaderelection/ordinal.go

```diff
@@ -27,20 +27,17 @@ import (
 // should be set to the pod name via the downward API.
 const controllerOrdinalEnv = "CONTROLLER_ORDINAL"
 
-// ParseControllerOrdinal returns the parts of the StatefulSet's pod name.
-func ParseControllerOrdinal() (string, uint64, error) {
-	v := os.Getenv(controllerOrdinalEnv)
+// ControllerOrdinal tries to get ordinal from the pod name of a StatefulSet,
+// which is provided from the environment variable CONTROLLER_ORDINAL.
+func ControllerOrdinal() (int, error) {
+	v, ok := os.LookupEnv(controllerOrdinalEnv)
+	if !ok {
+		return 0, fmt.Errorf("%s envvar is not set", controllerOrdinalEnv)
+	}
```

Comment:
Can we dedup the implementation with `newStatefulSetConfig`?

Reply:
done.